requirements.txt

```diff
@@ -4,10 +4,10 @@
 #
 #    tox -e pip-compile
 #
-ansible-modules-hashivault==3.9.4 \
```

Comment:
These changes shouldn't be part of this commit and are likely not tested, please revert.

Reply:
fixed

Reply:
No, only reverted `docs/requirements.txt` (whose update I don't really object against), not `/requirements.txt`, which is more impactful (and not used when building the docs, which is all that's tested here).